op-e2e/opgeth/op_geth_test.go

```diff
@@ -1099,3 +1099,123 @@ func TestFjord(t *testing.T) {
 		})
 	}
 }
+
+func TestIsthmus(t *testing.T) {
+	tests := []struct {
+		name            string
+		isthmusTime     hexutil.Uint64
+		activateIsthmus func(ctx context.Context, t *testing.T, opGeth *OpGeth)
+	}{
+		{name: "ActivateAtGenesis", isthmusTime: 0, activateIsthmus: func(ctx context.Context, t *testing.T, opGeth *OpGeth) {}},
```

Comment:
Can we also add a pre-isthmus test? To assert that the calling those precompiles result in the default behavior for an account with no code, i.e. no revert; 0x1 result.

Reply:
added! Seems to give an empty return value which I believe is correct - 0x1 is the status I'm assuming?

Reply:
Yeah I meant the status. Looks good.